transports/sqs/configuration-options_visibility-timeout_sqs_[7.3,).partial.md

```diff
@@ -0,0 +1,87 @@
+## Message visibility
+
+To prevent messages from being reprocessed while a handler is still executing, the transport automatically renews the message visibility timeout during processing. This is especially important for long-running handlers where the processing time may exceed the original visibility timeout.
```

Comment:
```suggestion
To prevent messages from being picked up by other consumers while a handler is still executing, the transport automatically renews the message visibility timeout during processing. This is especially important for long-running handlers where the processing time may exceed the original visibility timeout.
```

Reply:
I am not sure if I'm a fan of "by other consumers" because parallel fetching on the same consumer can also lead to reprocessing technically. I think the unspecific reprocessed is better here.